pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -137,7 +138,15 @@ func Execute(configFile io.Reader) {
 	app.Config.HTTP.Headers.Set("X-Registry-Supports-Signatures", "1")
 
 	app.RegisterHealthChecks()
-	handler := alive("/", app)
+	handler := http.Handler(app)
```

Comment:
@smarterclayton what I was saying is that HEAD will cause a write to blob store (we start mirroring and uploading to S3 in background, bypassing the API and rate-limiter).

@dmage or @legionus can explain it better, seems like i am a proxy here :) maybe you guys should talk